setup.py

```diff
@@ -24,7 +24,7 @@
 install_requires = [
     'diff-match-patch',
     'Django>=2.0',
-    'tablib[all]>=0.14.0',
+    'tablib[html,odf,xlsx,yaml]>=0.14.0',
```

Comment:
@claudep I like these defaults, but maybe we should also add `xls` as this have always been installed by `django-import-export` and it would be a surprise for users to have this change in a patch release. What do you think?

Reply:
I'd say we add it now, and drop it on a minor release.